code/modules/recycling/conveyor2.dm

```diff
@@ -390,7 +390,7 @@ GLOBAL_LIST_INIT(conveyor_switches, list())
 	switch(action)
 		if("slowFactor")
 			var/value = text2num(params["value"])
-			if (value!=null)
+			if(value!=null)
```

Comment:
```suggestion
			if(value != null)
```

Reply:
This PR is gonna be sticking exclusively to the spacing for conditionals, not the operators, because theres like 5000+ cases of those. Out of scope.